### ISSUE TITLE
Make seed input a set of discrete input boxes

### ISSUE THREAD

**Issue by gcharang** [TokelPlatform/tokel_app#113]
In both address gen flow and restore flow, when asking a user to input seed words, make the user enter each word in a different box. This way, missing/extra space user errors can be eliminated. the inputs can be trimmed, validated against word list, etc., and can be used inside the app to derive the actual seed phrase

**Comment by CHMEX** [comment]
Additionally the way it works now to enter the seed is very convenient and works for me as ma matter of fact I love the simplicity.
However people will lose their funds with this. As they will enter 2022 and think it's the password for the wallet. when it is actually the seed.

**Comment by gcharang** [comment]
@CHMEX you mean that the current UI is good for you. but, you think that newbies might get confused and will lose money right?

**Comment by CHMEX** [comment]
Yes correct for myself its perfect just to enter the wif and it works.
However new people will think its a password to protect the wallet so they
use 2020 with is the seed.
Same we had in agama where 300k was deposited a famous address seed was
pretty weak.....

**Comment by piggydoughnut** [comment]
ah good point, didnt think about this kind of use case at all. Thx for mentioning it.
